### PR TITLE
Make the title optional

### PR DESCRIPTION
**Pull request by wlawski**

### CODE REVIEW

File: src/multiple_choice/template.py [hunk 1]
```diff
@@ -361,8 +361,8 @@
 aio_model = "AllInOne (kprim, mc, sc)"
 aio_card = "AllInOne (kprim, mc, sc)"
 aio_fields = {
-    "title": "Title",
```
Comment (3ter) [inline src/multiple_choice/template.py:364]: I do think that's a legit improvement to be made. Can you please explain how changing the order making this field optional?

Comment (wlawski) [reply src/multiple_choice/template.py:364]: Only the first field must not be empty (usually, it is a question): https://github.com/ankitects/anki/blob/main/qt/aqt/addcards.py#L171
You can find dupeOrEmpty() function return values explanation here: https://github.com/ankitects/anki/blob/3ba08d1189999c38f983288e980f39494c1c0057/pylib/anki/notes.py#L182

Comment (3ter) [reply src/multiple_choice/template.py:364]: Ah, perfect. Thanks for the reference!

**Review by 3ter — APPROVED** [review]

**Comment by 3ter** [comment]
It probably makes sense to change this to `0` then:
https://github.com/zjosua/anki-mc/blob/ed0536443724cb32dca7293638c11377d3963f1c/src/multiple_choice/template.py#L507

**Comment by wlawski** [comment]
> It probably makes sense to change this to `0` then:
> https://github.com/zjosua/anki-mc/blob/ed0536443724cb32dca7293638c11377d3963f1c/src/multiple_choice/template.py#L507

Yes, indeed. I have just updated it. Thanks!

**Pull request merged**